contrib_bots/lib/spotify.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+from future import standard_library
+standard_library.install_aliases()
+import json, urllib.request, urllib.parse, urllib.error
+class SpotifyHandler(object):
+    '''
+    This plugin will allow users to directly search
+    for song, artists, and albums using the Spotify API.
+    It can be called using `@spotify song/artist/album`
```

Comment:
@trueskawka searching for a artist won't give the 10 tracks from the artist. It will give you a list of different artists matching with your query.

If you know the song then you search for it. The search of a track returns a list of tracks with the artist who composed it.